toolchain/check/decl_name_stack.cpp

```diff
@@ -206,6 +206,11 @@ static auto PushNameQualifierScope(Context& context,
   // parameter scope.
   context.scope_stack().PopIfEmpty();
 
+  // When declaring a member of a generic, resolve the specific self.
```

Comment:
```suggestion
  // When declaring a member of a generic, resolve the specific's self.
```

Reply:
What I mean here is the specific that is the generic's self -- the thing we used to call the "self instance". Maybe
```suggestion
  // When declaring a member of a generic, resolve the self specific.
```
would work?

Reply:
Yeah, I'd just sub "instance" with "specific" for how you'd phrase it.